changelog.d/3473.fixed

```diff
@@ -0,0 +1 @@
+Windows UEFI
```

Comment:
Not very descriptive, I think giving a little more would be helpful.

Reply:
I redistributed the description more correctly between “added”, “changed” and “corrected”.